test/Microsoft.AspNetCore.Mvc.Razor.Extensions.Test/TestFiles/IntegrationTests/CodeGenerationIntegrationTest/PageWithNamespace_DesignTime.codegen.cs

```diff
@@ -1,3 +1,5 @@
+// <auto-generated/>
```

Comment:
Does this appear after the checksum pragma? That has to be the first line in the generated code.